models/user.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+const { Model } = require("sequelize");
+
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    static associate(models) {}
+  }
+  User.init(
+    {
+      id: {
+        field: "user_uuid",
+        type: DataTypes.UUID,
+        primaryKey: true,
+      },
+      firstName: {
+        field: "firstname",
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      lastName: {
+        field: "lastname",
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      email: {
+        field: "email",
+        type: DataTypes.STRING,
+      },
+      subjectId: {
+        field: "subjectid",
+        type: DataTypes.STRING,
+      },
+      password: {
```

Comment:
The field is `hashedpassword` and the attribute should also be `hashedPassword`.

You can then add a separate "virtual" `password` attribute that will be ephemeral and only used to capture the plaintext password before hashing it and storing it in the hashedPassword column.

https://sequelize.org/master/manual/getters-setters-virtuals.html#virtual-fields